src/components/Person/Person.jsx

```diff
@@ -1 +1,9 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
```

Comment:
Destructure the person above
```suggestion
    <h2 className="Person__name">{`My name is ${name}`}</h2>
```